components/google_cloud/sources/common/bigquery.js

```diff
@@ -0,0 +1,130 @@
+const { BigQuery } = require("@google-cloud/bigquery");
+const { GoogleAuth } = require("google-auth-library");
+const chunk = require("lodash/chunk");
```

Comment:
Isn't the package `lodash.chunk`? https://www.npmjs.com/package/lodash.chunk

This might work, but I think it might pull in all of `lodash` (not a big deal either way).

Reply:
Apparently that approach is being deprecated in favour of using something like `lodash/chunk`:
https://lodash.com/per-method-packages

Reply:
Interesting, did not know that. Thanks!